README.md

```diff
@@ -5,28 +5,32 @@
 [![GHA](https://img.shields.io/github/v/tag/iterative/setup-cml?label=GitHub%20Actions&logo=GitHub)](https://github.com/iterative/setup-cml)
 [![npm](https://img.shields.io/npm/v/@dvcorg/cml?logo=npm)](https://www.npmjs.com/package/@dvcorg/cml)
 
-**What is CML?** Continuous Machine Learning (CML) is an open-source library for
-implementing continuous integration & delivery (CI/CD) in machine learning
-projects. Use it to automate parts of your development workflow, including model
-training and evaluation, comparing ML experiments across your project history,
-and monitoring changing datasets.
+**What is CML?** Continuous Machine Learning (CML) is an open-source CLI tool
+for implementing continuous integration & delivery (CI/CD) with a focus on
+MLOps. Use it to automate parts of development workflows — including machine
+provisioning; model training and evaluation; comparing ML experiments across
+project history, and monitoring changing datasets.
```

Comment:
💅  Curious about the use of `;` mixed with `,`.

Reply:
As per [the MLA style guide](https://style.mla.org/serial-commas-and-semicolons), semicolons can be used to delimit items that already contain commas, but, unlike the revewed excerpt, they seem to recommend `; and` instead of `, and` before the last item; the latter is pretty common in Spanish, so I didn't give it a second thought.

The first time I reviewed this text, what shocked me is that none of the list items had a comma nor a conjunction that justified the use of semicolons. 🤔